handbook/company/communications.md

```diff
@@ -213,6 +220,7 @@ Before spending any money on an offsite, inviting people, booking travel, or oth
     - tentative dates
     - a lean budget (including links and street address of lodging and event spaces, estimated airfare, and spending for other food or accomodations)
     - a detailed agenda of how time will be spent
+    - BizOps has a [template (private google doc)](https://docs.google.com/spreadsheets/d/1oe1wAFzerXlKghZaZtdlA7QrWBv9iZmHThlFYqfVzRQ/edit#gid=1927695119) you can use to guide your planning.
```

Comment:
Nice template, I like it!

```suggestion
    - Use [this offsite template (private google doc)](https://docs.google.com/spreadsheets/d/1oe1wAFzerXlKghZaZtdlA7QrWBv9iZmHThlFYqfVzRQ/edit#gid=1927695119) as a starting point to guide your planning.
```

Or update "Write a google doc with your proposed plan" with a link

Reply:
because then it would come first too, so people are more likely to use it